mono/mini/mini-runtime.c

```diff
@@ -4657,7 +4657,7 @@ register_icalls (void)
 		register_icall (mono_fload_r4, "mono_fload_r4", "double ptr", FALSE);
 		register_icall (mono_fstore_r4, "mono_fstore_r4", "void double ptr", FALSE);
 		register_icall (mono_fload_r4_arg, "mono_fload_r4_arg", "uint32 double", FALSE);
-		register_icall (mono_isfinite, "mono_isfinite", "uint32 double", FALSE);
+		register_icall (mono_isfinite, "mono_jit_icall_isfinite", "uint32 double", FALSE);
```

Comment:
Should be `register_icall (mono_jit_icall_isfinite, "mono_jit_icall_isfinite", ...)`

Reply:
Good catch.

Reply:
I'd also be open to a new naming scheme. mono_math_isinf(), mon_c99_isinf, or something. I was reluctant to rename this function, but discovered it late, and think I picked a pretty good name for the others.

We could also expose mono_isinfinite_double here, the new function to replace the old, but they seem just ever so slightly different in intent -- icall vs. c99 wrappers.